viewer/src/utilities/index.ts

```diff
@@ -14,6 +14,7 @@ export { LocalModelIdentifier, CdfModelIdentifier } from './networking/types';
 export { createOffsetsArray } from './arrays';
 export { Box3 } from './Box3';
 export { HtmlOverlayTool as HtmlOverlayHelper } from '../tools/HtmlOverlayTool';
+export { ExpandAssetTool } from '../tools';
```

Comment:
This should not be exported here - it belongs to `tools.ts` (I've removed the `HtmlOverlayTool` export in a separate PR #1383 )